crates/router/src/core/payout_link.rs

```diff
@@ -300,3 +343,41 @@ pub async fn filter_payout_methods(
     }
     Ok(response)
 }
+
+pub async fn check_currency_country_filters(
+    payout_method_filter: Option<&PaymentMethodFilters>,
+    request_payout_method_type: &api_models::payment_methods::RequestPaymentMethodTypes,
+    currency: &common_enums::Currency,
+    country: &Option<enums::CountryAlpha2>,
+) -> errors::RouterResult<Option<bool>> {
```

Comment:
Why can't we return bool here, instead of Option<bool>?

Reply:
If there is no filter available for connector-> payout method configuration we will still show the payout method by default. We can't remove payout method just because filters are not declared for that.